modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/server/organization/management/v1/OrganizationManagementSuccessTest.java

```diff
@@ -111,9 +113,6 @@ public class OrganizationManagementSuccessTest extends OrganizationManagementBas
 
     protected OAuth2RestClient restClient;
 
-    private static final int NUM_OF_ORGANIZATIONS_FOR_PAGINATION_TESTS = 20;
```

Comment:
Any specific reason to remove this constants?

Reply:
Moved it to the Base Class.